src/main/java/com/googlecode/lanterna/gui2/dialogs/ActionListDialog.java

```diff
@@ -35,6 +35,16 @@
             TerminalSize actionListPreferredSize,
             boolean canCancel,
             List<Runnable> actions) {
+        this(title, description, actionListPreferredSize, canCancel, true, actions);
```

Comment:
No need to make a constructor override, just amend the existing one

Reply:
I'm a bit confused.  Is compatibility a non-issue?

Reply:
Package-private, so I'm ok with changing it (as long as lanterna compiles)

Reply:
Oh, right, I missed that.  Sorry for my noise.